hs_core/views/__init__.py

```diff
@@ -64,6 +65,33 @@
 logger = logging.getLogger(__name__)
 
 
+class HydroshareWebServices():
+
+    def __init__(self, services_url, api_token, timeout):
+        self.services_url = services_url
+        self.timeout = timeout
+        self.session = requests.Session()
+        self.session.headers.update(
+            {"Authorization": " ".join(("Token", str(api_token)))}
+        )
+
+    def update(self, res_id):
+        rest_url = str(self.services_url) + "/update-services/?res_id=" + str(res_id)
+        try:
+            return self.session.post(rest_url, timeout=self.timeout)
+        except requests.exceptions.RequestException as requests_exception:
+            return requests_exception
+        except ValueError as requests_exception:
+            return requests_exception
+
+
+web_services = HydroshareWebServices(
+    services_url="http://localhost:8080/apps/hydroshare-web-services-manager/api",
+    api_token="932aebc70b5924dd31da63f8d80e7c62db18c02c",
+    timeout=20
```

Comment:
Values for these variables ```services_url```, ```api_token```, and ```timeout```  need to go to settings.py or local_settings.py. They should not be hardcoded in the code.

Reply:
@kjlippold Please watch this PR for any implications to your activity after it is merged into develop https://github.com/hydroshare/hydroshare/pull/3082